uk.co.compucorp.civicrm.hrcore/CRM/HRCore/CMSData/UserAccount/Drupal.php

```diff
@@ -69,6 +69,6 @@ public function enable($contactData) {
   public function isUserDisabled($contactData) {
     $user = $this->getUser($contactData);
 
-    return $user->status;
+    return !$user->status ? true : false;
```

Comment:
Why didn't the test catch this? Shouldn't the test be updated so that the error doesn't happen again?